prac_06/programming_language.py

```diff
@@ -0,0 +1,17 @@
+"""CP1404/CP5632 Practical - Programming Language Intermediate Exercise"""
+
+class ProgrammingLanguage:
+
+    def __init__(self, name, typing, reflection, year):
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def is_dynamic(self):
+        if self.typing.lower() == "dynamic":
```

Comment:
return self.typing == 'Dynamic' (Code that I used)
I guess the if and true statements are not necessary on programming_language.py since it will be handled by the code done on languages.py.